package.json

```diff
@@ -94,6 +94,7 @@
     "eslint": "^8.28.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-header": "^3.1.1",
+    "eslint-plugin-no-only-tests": "^3.3.0",
```

Comment:
Late to the party, but why using this plugin instead of the Jest one. it offers this rule but also other useful for us too: https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-focused-tests.md

Reply:
good suggestion, following up here https://github.com/cloudscape-design/components/pull/3301